src/control/server/ctl_ranks_rpc_test.go

```diff
@@ -691,10 +647,8 @@ func TestServer_CtlSvc_ResetFormatRanks(t *testing.T) {
 		"instances stay stopped": {
 			req:        &ctlpb.RanksReq{Ranks: "0-3"},
 			startFails: true,
-			expResults: []*sharedpb.RankResult{
-				{Rank: 1, State: msStopped, Errored: true},
-				{Rank: 2, State: msStopped, Errored: true},
-			},
+			timeout:    500 * time.Millisecond,
```

Comment:
Could set a shorter timeout than this, I think. Are we racing against something?

Reply:
yes some of them can be shorter but I didn't think this was an unreasonable value